core/src/apps/management/recovery_device/backup_types.py

```diff
@@ -0,0 +1,23 @@
+from trezor.messages import BackupType
+
+# possible backup types based on the number of words
+TYPES = {
+    12: [BackupType.Bip39],
+    18: [BackupType.Bip39],
+    24: [BackupType.Bip39],
+    20: [BackupType.Slip39_Basic, BackupType.Slip39_Advanced],
+    33: [BackupType.Slip39_Basic, BackupType.Slip39_Advanced],
+}
+
+
+def get(word_count: int) -> int:
```

Comment:
Thanks. Btw how can I type that it is a list of three possible values BackupType.Bip39, BackupType.Slip39_Basic, BackupType.Slip39_Advanced? Is it `List[Union[BackupType.Bip39, BackupType.Slip39_Basic, BackupType.Slip39_Advanced]]`?

Reply:
Yes, but it would be better to declare the type somewhere (`mnemonic.py`?):

```
if False:
	MnemonicType = Union[BackupType.Bip39, BackupType.Slip39_Basic, BackupType.Slip39_Advanced]
```

Reply:
Nice, will do. Thanks.